server/tracer/src/main/java/org/apache/accumulo/tracer/TraceServer.java

```diff
@@ -169,23 +170,25 @@ public void span(RemoteSpan s) throws TException {
         if (timeMutation != null)
           writer.addMutation(timeMutation);
       } catch (MutationsRejectedException exception) {
-        log.warn("Unable to write mutation to table; discarding span. set log level to DEBUG for span information and stacktrace. cause: " + exception);
+        log.warn("Unable to write mutation to table; discarding span. set log level to DEBUG for span information and stacktrace. cause: {}", exception,
```

Comment:
I pulled these changes from a patch supplied by Christopher and I think some of these got snagged with them.  I guess I can create a ticket with something like "utilize slf4j"?  A quick search in JIRA didn't reveal any tickets like that...